.github/workflows/meson.yml

```diff
@@ -44,6 +44,7 @@ jobs:
 
           - name: Ubuntu
             os: ubuntu-latest
+            args: '-Db_sanitize=address,undefined'
```

Comment:
How does this meson option deal with clang’s UBSAN either never setting a non-zero exit code, or not printing a report about _what_ went wrong?

Reply:
> Actually this decreases build diversity since all Ubuntu-based autotools runners use ASAN and UBSAN, but this seems hardly problematic

Fair. But still ASAN is generally good to run either way. I mean if it works with ASAN it will work without it too.

> How does this meson option deal with clang’s UBSAN either never setting a non-zero exit code, or not printing a report about what went wrong? (ref 

Meson sets UBSAN_OPTIONS and ASAN_OPTIONS that are appropriate for running tests. Specifically it sets `halt_on_error=1`.

Probably ghci.yml could also set `UBSAN_OPTIONS=halt_on_error=1` this will alow the print of the report.

Reply:
Sure, if LLVM/CLang now actually has a way to do this that will be preferable (we already set `-fno-sanitize-recover` anyway, but in the past clang’s UBSAN used to ignore all exitcode runtime flags)